components/global_header/hooks.tsx

```diff
@@ -42,10 +42,6 @@ export function useClickOutsideRef(ref: MutableRefObject<HTMLElement | null>, ha
 export const useCurrentProductId = (products?: ProductComponent[]): string | null => {
     const location = useLocation();
 
-    if (!products) {
-        return null;
-    }
```

Comment:
This duplicate check moves to https://github.com/mattermost/mattermost-webapp/pull/11147/files#diff-306df37733404f27ab6dedef74b04b3ab268fb9fa9c1eb5ab7b063af88dff177R6